Project.toml

```diff
@@ -1,9 +1,6 @@
 name = "InclusiveScans"
 uuid = "ef74772d-374f-5edb-a43a-0dbb31824c9d"
-authors = [
-    "Uwe Hernandez Acosta",
-    " Simeon Ehrig",
-]
+authors = ["Uwe Hernandez Acosta", " Simeon Ehrig"]
```

Comment:
```suggestion
authors = ["Uwe Hernandez Acosta", "Simeon Ehrig"]
```

Reply:
Nitpicking, but fine 😅

Reply:
I knew it! It's Anton's fault 😆

Reply:
Vile slander D:
I'm completely innocent, I would never nitpick!